model/lib/schema.mustache

```diff
@@ -17,4 +17,7 @@ exports.filter = (name, params) => {
   }, {});
 };
 
-exports.validate = jsen(schema);
+exports.getValidator = (name) => {
```

Comment:
I don't think this is much of a problem. Redbeard once scaffolded has generated your files and they are then effectively yours to do what you will. Upgrading redbeard is really not going to be a simple task as once you build on top of it there isn't really any simple path to bring in the new changes without blowing away your own work.